Backend/Core/APIServer.py

```diff
@@ -42,6 +43,9 @@ def _checkData(self, requestName, data):
                 "latitude",
                 "longitude",
                 "number"
+            ],
+            "GeocodingAddress": [
+                "q"
```

Comment:
What is "q" ? What does it stand for? A param which is just called q isn't really meaningfull.

Reply:
q stands for query, it’s from the Geocoding API, free form query string to search for.

Reply:
> What is "q" ? What does it stand for? A param which is just called q isn't really meaningfull.

q stands for query, it’s from the Geocoding API, free form query string to search for.